src/components/FeatureViews/Accounts/TabComponents/InstructorCourses.js

```diff
@@ -8,35 +8,53 @@ import Paper from "@material-ui/core/Paper";
 import Typography from "@material-ui/core/Typography";
 import UnconfirmIcon from "@material-ui/icons/Cancel";
 import Moment from "react-moment";
+import moment from "moment";
 
 import gql from "graphql-tag";
 import { useQuery } from "@apollo/react-hooks";
+import Loading from "components/OmouComponents/Loading";
+import { DayAbbreviation } from "utils";
 
 export const GET_INSTRUCTOR_ENROLLMENTS = gql`
   query InstructorEnrollments($instructorID: ID!) { 
     courses(instructorId: $instructorID) {
       id
-      dayOfWeek
       description
       startDate
-      startTime
       endDate
-      endTime
       isConfirmed
       title
+      availabilityList {
+        id
+        dayOfWeek
+        endTime
+        startTime
+      }
     }
   }
 `;
 
+const checkTimes = (courseTimes) => {
+  let start = courseTimes[0].startTime;
+  let end = courseTimes[0].endTime;
+
+  for (let course of courseTimes) {
```

Comment:
this doesn't make sense - shouldn't it be `let availability of courseAvailabilities` ?